### PR TITLE
Concert Bid Adapter: Update localStorage name-spacing for Concert UID

### PR DESCRIPTION
**Pull request by BrettBlox**
## Type of change
- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
- adds check for sharedId falls back to existing method of none exists
- swaps out localStorage key `c_uid` with `vmconcert_uid` to avoid potential collisions with vendors
- fixes broken tests related to localStorage
- updates test with new localStorage key name

### CODE REVIEW

**Comment by patmmccann** [comment]
In general, this is bad netizenship. You generate a uuid and set it in the first party domain. Why not check for pubcommonid, which is the exact same thing and use that and fallback to this when missing?

**Comment by patmmccann** [comment]
@jdwieland8282 as fyi; as these examples start to proliferate, should we crack down on them?

**Comment by jdwieland8282** [comment]
Thanks for surfacing this, yes we should, absolutely.

**Comment by jdwieland8282** [comment]
@BrettBlox what are your reservations about checking for and using sharedid? Using a universal open source id namespace (pubcid.org) has many advantages over a bespoke one.

**Comment by BrettBlox** [comment]
@jdwieland8282 I will look into it. Thanks.

**Comment by BrettBlox** [comment]
Updated to check for sharedId.

File: modules/concertBidAdapter.js [hunk 1]
```diff
@@ -188,10 +188,24 @@ function getUid(bidderRequest) {
     return false;
   }
 
-  const CONCERT_UID_KEY = 'c_uid';
+  const sharedId = deepAccess(bidderRequest, 'userId._sharedid.id');
 
+  if (sharedId) {
+    return sharedId;
```
Comment (patmmccann) [inline modules/concertBidAdapter.js:194]: Awesome, ty

**Review by mmoschovas — APPROVED** [review]

**Pull request merged**